lib/data/data.dart

```diff
@@ -6,6 +8,79 @@ class UserDetail {
   UserDetail({required this.uid, this.phone, this.email});
 }
 
+enum PropertyTypes {
+  house,
+  apartment;
+
+  static PropertyTypes fromName(String name) =>
+      values.firstWhere((element) => element.name == name);
+}
+
+enum AustraliaStates {
+  //TODO: add more states for upcomming story
+  nsw('New South Wales'),
```

Comment:
Please move the strings into 
l10n, you can still access these strings from l10n using .current in the String object